README.md

```diff
@@ -22,7 +22,7 @@ To build this website, we wrote down 10 rules for how to make a low impact websi
 
 ## Setup
 
-The Low Impact Website uses Shopify's [Storefront API](https://shopify.dev/docs/storefront-api) to fetch products and basic ecommerce functionality.
+The Low Impact Website uses Shopify's [Storefront API](https://shopify.dev/docs/storefront-api) to fetch products and basic e-commerce functionality.
```

Comment:
I don't think it matters much whether we use "e-commerce" or "ecommerce" as long as we are consistent. 

Shopify uses "ecommerce", so we should stick with that as well.